### PR TITLE
added new loss function

### PR DESCRIPTION
**Pull request by shreyasmisra**
computes loss b/w sin and cosine components

### CODE REVIEW

File: astronomy_cv_takehome/loss.py [hunk 1]
```diff
@@ -5,17 +5,32 @@ class CustomLoss(nn.Module):
...
 
         #TODO(SM): Implement the coefficients as parameters of the model
 
-    def _yaw_loss(self, yaw_true, yaw_pred):
-        pass
+    def _yaw_loss_fn(self, yaw_pred, yaw_true):
+        """
+        Args:
+            yaw_true: ground truth yaw
+            yaw_pred: sin and cos of the yaw
```
Comment (shreyasmisra) [inline astronomy_cv_takehome/loss.py:18]: Not predicting the sine and cosine of the yaw anymore. Since I'm limited to 5 model outputs from the network

File: astronomy_cv_takehome/train.py [hunk 2]
```diff
@@ -117,10 +110,11 @@ def main():
                         default='mobilenet', type=str, 
                         choices=("mobilenet", "regnet", "mnasnet", "shufflenet"))
     parser.add_argument('--scheduler', default='plateau', type=str, choices=("plateau", "step", "cosine"))
-    parser.add_argument('--optimizer', default='Adam', type=str, choices=("Adam", "SGD"))
     parser.add_argument('--num-dense', type=int, default=256)
     parser.add_argument('--init-weights', action='store_true')
     parser.add_argument('--activation', type=str, default='relu', choices=('relu', 'sigmoid', 'tanh'))
+    parser.add_argument('--reg-coeff', type=float, default=1.0)
+    parser.add_argument('--yaw-coeff', type=float, default=2.0)
```
Comment (shreyasmisra) [inline astronomy_cv_takehome/train.py:117]: Probably should try both as 1 first.
